docs/conf.py

```diff
@@ -80,7 +80,7 @@
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'UPDATE.md']
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "sphinx"
+pygments_style = 'sphinx'
```

Comment:
lets fo these changes in separate PR so it is easier to follow what is the main change/focus of this PR